guides/source/testing.md

```diff
@@ -89,26 +107,36 @@ class ArticleTest < ActiveSupport::TestCase
 end
 ```
 
-A line by line examination of this file will help get you oriented to Rails testing code and terminology.
+A line by line examination of this file will help get you oriented to Rails
+testing code and terminology.
 
 ```ruby
 require "test_helper"
 ```
 
-By requiring this file, `test_helper.rb`, the default configuration to run our tests is loaded. We will include this with all the tests we write, so any methods added to this file are available to all our tests.
+By requiring this file, `test_helper.rb`, the default configuration to run our
+tests is loaded. We will include this with all the tests we write, so any
+methods added to this file are available to all our tests.
```

Comment:
A possible rewording to eliminate "our" and "we". I think it's okay to have those pronouns, though in this case not necessary. Feel free to ignore, just a minor suggestion:

```suggestion
Requiring the file, `test_helper.rb`, loads the default configuration to run tests. All methods added to this file are also available in tests by including this file.
```